src/sentry/static/sentry/app/views/organizationIntegrations.jsx

```diff
@@ -168,11 +166,15 @@ export default class OrganizationIntegrations extends AsyncView {
                           </small>
                         </td>
                         <td style={{width: 60}}>
-                          <button
-                            onClick={this.deleteIntegration.bind(this, integration)}
-                            className="btn btn-default btn-xs">
-                            <span className="icon icon-trash" />
-                          </button>
+                          <Confirm
+                            message={t(
+                              'Are you sure you want to remove this integration?'
+                            )}
+                            onConfirm={this.deleteIntegration.bind(this, integration)}>
```

Comment:
I personally think it reads better to write `onConfirm={()=>this.deleteIntegration(integration)}>` thoughts?

Reply:
It is more intuitive esp since people are probably not as familiar with `bind`. Wonder if there's an eslint rule for this....